scripts/exgfs_wave_post_gridded_sbs.sh

```diff
@@ -416,20 +416,20 @@ source "${USHgfs}/preamble.sh"
         echo ' '
         set_trace
         err=5; export err;${errchk}
-        exit $err
+        exit "$err"
       fi
-      if [ $FHMAX_HF_WAV -gt 0 ] && [ $FHOUT_HF_WAV -gt 0 ] && [ $fhr -lt $FHMAX_HF_WAV ]; then
+      if [ $FHMAX_HF_WAV -gt 0 ] && [ $FHOUT_HF_WAV -gt 0 ] && [ "$fhr" -lt $FHMAX_HF_WAV ]; then
         FHINCG=$FHOUT_HF_WAV
       else
         FHINCG=$FHOUT_WAV
       fi
       fhrg=$((fhr+FHINCG))
     fi
-    echo $fhrg
+    echo "$fhrg"
 
     fhr=$fhrg #loop with out_grd stride
```

Comment:
This is more stuff to calculate the next hour to run when it was a loop and is no longer needed.

Reply:
@WalterKolczynski-NOAA Modified `gfs_tasks.py` and did your suggested changes to `exgfs_wave_post_gridded_sbs.sh`. Ran `GEFS` and `GFS` tests on `HERCULES` and they all passed. Did a check on output files and see no difference (between `develop` and current branches).  Anymore suggestions?